app/src/main/java/org/oppia/android/app/mydownloads/downloads/DownloadsFragmentPresenter.kt

```diff
@@ -0,0 +1,205 @@
+package org.oppia.android.app.mydownloads.downloads
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ArrayAdapter
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import org.oppia.android.R
+import org.oppia.android.app.fragment.FragmentScope
+import org.oppia.android.app.recyclerview.BindableAdapter
+import org.oppia.android.databinding.DownloadsFragmentBinding
+import org.oppia.android.databinding.DownloadsSortbyBinding
+import org.oppia.android.databinding.DownloadsTopicCardBinding
+import javax.inject.Inject
+
+const val DELETE_DOWNLOAD_TOPIC_DIALOG_TAG =
+  "DownloadsTopicDeleteDialogFragment.delete_download_topic_dialog_tag"
+const val ADMIN_PIN_CONFIRMATION_DIALOG_TAG =
```

Comment:
Suggest defining them in `DownloadsFragment` companion object and making them internal.

Reply:
I think making them internal keeps them accessible only in DownloadsFragment

Reply:
I think you will be able to access them via:
 `DownloadsFragment.ADMIN_PIN_CONFIRMATION_DIALOG_TAG `
anywhere you want.

Reply:
You will need to make this change at multiple places in this PR.

Reply:
done